apps/passport-server/src/services/generic-issuance/pipelines/PODPipeline/PODPipeline.ts

```diff
@@ -79,6 +87,9 @@ export class PODPipeline implements BasePipeline {
     cacheService: PersistentCacheService
   ) {
     this.eddsaPrivateKey = eddsaPrivateKey;
+    const privateKeyBytes = decodePrivateKey(eddsaPrivateKey);
+    const unpackedPublicKey = derivePublicKey(privateKeyBytes);
+    this.eddsaPublicKey = encodePublicKey(unpackedPublicKey);
```

Comment:
Are you re-encoding here to canonicalize the string format?
Reasonable thing to do, though I don't understand why it's necessary here in particular.

Reply:
This is deriving the public key from the private key. I'm not sure how else this could be done.

Reply:
However, since there's no need to create entries with the signer's public key, I can remove this.

Reply:
Oh, you're right, I misread what was going on here.  I thought you were unpacking and re-packing the same key.